docs/reference/3-admin-api/installation/admin-api-2x-for-odsapi-7x/admin-api-2x-iis-installation-manual.md

```diff
@@ -0,0 +1,416 @@
+
+For the certificate, use "Ed-Fi-ODS" as the friendly name and make sure the
+certificate store is set to "Personal". Click OK. We will use this certificate
+in an upcoming step.
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v2/image2019-7-17_14-21-50.png)
+
+### **Step 6. Create Necessary Application Pools**
+
+Back in the IIS Manager main page, expand the server/machine on the left sidebar
+and click on Application Pools.
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v2/image2019-7-17_15-53-53.png)
+
+Click on "Add Application Pool..." on the Actions bar to the right, enter
+"Ed-Fi" as the name. Click OK.
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v2/image2019-7-17_15-57-20.png)
+
+Once that is created, click on the "Ed-Fi" application pool and select "Advanced
+Settings..." on the Actions bar to the right. Change the Start Mode to
+"AlwaysRunning".
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v2/image2019-7-17_16-1-46.png)
+
+This next bit is optional if you want to use an app pool identity. If you would
+like to use the default "ApplicationPoolIdentity", then you can skip this
+section.
+
+In the same Advanced Settings window, click on the browse icon under Process
+Model > Identity. Choose the custom account option and click "Set...". When
+setting the credentials, you can simply use the username and password that you
+use to log in to Windows. If you need to include the app pool domain in the
+username, then the username can look something like this: "localhost\\username",
+where "localhost" is the app pool domain. Once you've entered the correct
+credentials, click OK on all screens until you're back to the main Application
+Pools page.
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v2/image2019-7-17_16-8-37.png)
+
+After that's done, we'll repeat the exact same steps in this section, but enter
+"Admin API" as the Application Pool name. We should now have 2 app pools.
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v2/image2022-8-12_11-59-40.png)
+
+### **Step 8. Create Admin API Website**
+
+On the left sidebar, we'll right-click on the newly created "Ed-Fi" website,
+select "Add Application...", and perform the following:
+
+1. Enter "Admin API" for the Alias.
+2. For the Application pool, select the Admin API app pool.
+3. The Physical path should be set to the path of the Admin API folder (located
+    in directory with Admin API source files).
+4. Hit OK.
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v2/image2019-7-17_16-31-47.png)
```

Comment:
If this is the same screenshot as on the other pages, you could reuse the same image. I'm not trying to force the issue though, since the image is in Azure rather than in the repository.